certified-connectors/Connect Bridge/apiDefinition.swagger.json

```diff
@@ -297,6 +321,18 @@
         "description": "This action closes a connection. All connections must be closed",
         "operationId": "CloseConnection",
         "parameters": [
+          {
+            "name": "format",
+            "in": "query",
+            "required": true,
```

Comment:
Please confirm that the existing flows for this connector will not break from this change.

The required parameter with default value has been added. ValuePath: operation/CloseConnection/parameter/format

Reply:
I confirm that the existing flows will not break from this change.

Actually this is a workaround because the policy stopped working for some reason.